### PR TITLE
PLT-5892 Raffle and chunked-deposit contracts

### PR DESCRIPTION
**Pull request by bwbush**
Added a raffle contract and a chunked-deposit contract to the `marlowe-contracts` package.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
        - Review required
        - [x] Substantial changes to code, test, or documentation
        - [ ] Change made to Marlowe validator (@bwbush and @palas must be included as reviewers)
        - Review not required
        - [ ] Minor changes to non-critical code, documentation, nix derivations, configuration files, or scripts
        - [ ] Formatting, spelling, grammar, or reorganization
    - [x] Reviewer requested

### CODE REVIEW

File: marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs [hunk 1]
```diff
@@ -249,7 +250,7 @@ runCreateCommand TxCommand{walletAddresses, signingMethod, tagsFile, metadataFil
         else do
           hPutStrLn stderr "Safety analysis found the following errors in the contract:"
           BS8.hPutStrLn stderr $ Yaml.encode safetyErrors
-    liftIO . print $ A.encode (A.object [("contractId", toJSON . renderTxOutRef $ contractId)])
+    liftIO . putStrLn . T.unpack . renderTxOutRef $ contractId
```
Comment (bwbush) [inline marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs:253]: This will break the tutorials, including the starter kit, so we'll need to update those.

File: nix/marlowe-cardano/compose.nix [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 let
   inherit (pkgs) z3 sqitchPg postgresql runCommand writeShellScriptBin writeText lib glibcLocales;
-  network = inputs.self.networks.preview;
+  network = inputs.self.networks.preprod;
```
Comment (bwbush) [inline nix/marlowe-cardano/compose.nix:5]: Why the change in networks?

Comment (paluh) [reply nix/marlowe-cardano/compose.nix:5]: Oh, accident.

File: marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs (hunk 1 above)
Comment (paluh) [reply marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs]: Another accident :-(
